miso-web/src/main/webapp/scripts/hot_pool.js

```diff
@@ -9,28 +9,41 @@ HotTarget.pool = {
   createColumns : function(config, create, data) {
     return [
         {
+          header : 'Pool Name',
+          data : 'name',
+          readOnly : true,
+          include : true,
+          unpackAfterSave : true,
+          unpack : function(pool, flat, setCellMeta) {
+            flat.name = pool.name;
+          },
+          pack : function(pool, flat, errorHandler) {
+            pool.name = flat.name;
```

Comment:
If the name is going to not be written back to the DB, then it should be dropped at every step. If it's copied, then it mysteriously vanishes at the next step, that's confusing. If it's dropped at every transfer point, then it's clear it wasn't meant to get saved.